pytorch_lightning/trainer/trainer.py

```diff
@@ -283,6 +283,9 @@ def __init__(
         self.shown_warnings = set()
 
         # init callbacks
+        # Declare attributes to be set in callback_connector on_trainer_init
+        self.checkpoint_callback: Union[ModelCheckpoint, bool] = checkpoint_callback
+        self.early_stop_callback: Optional[Union[EarlyStopping, bool]] = early_stop_callback
```

Comment:
https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/trainer/connectors/callback_connector.py#L44-L46

Reply:
dies this typing any effect?

Reply:
I don't think so since the trainer attributes aren't defined inside the trainer class/init